### ISSUE TITLE
Improve the output component

### ISSUE THREAD

**Issue by anshumanv** [anshumanv/evolutions-webapp#17]
Make a separate, good output component and render it for displaying all results.

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [ ] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.
